src/global_const.js

```diff
@@ -78,7 +82,7 @@ module.exports._ua = (function (u) {
             || u.indexOf("ipod") != -1 || (u.indexOf("android") != -1 && u.indexOf("mobile") != -1)
             || (u.indexOf("firefox") != -1 && u.indexOf("mobile") != -1) || u.indexOf("blackberry") != -1
     }
-})(window.navigator.userAgent.toLowerCase());
+})();
```

Comment:
`navigator` is accessible from either main or worker thread
https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope

I suggest
```js
function (u){
    return {/* snip */]
})(navigator.userAgent.toLowerCase() || "");
```

reason:
- remove browser dependency from function body (but not from module)
- an empty value as undefined-case argument (non-browser environment)
  - reduce maintenance cost `self.window` twice

NOTE: pls no touch Table detect code, can't device tests in this PR (not refactoring time)